graphstorm-processing/tests/test_converter.py

```diff
@@ -227,6 +227,14 @@ def test_convert_gsprocessing(converter: GConstructConfigConverter):
                     "feature_name": "rank_gauss2",
                     "transform": {"name": "rank_gauss", "epsilon": 0.1},
                 },
+                {
+                    "feature_col": ["num_citations"],
+                    "transform": {"name": "to_categorical", "mapping": {"1", "2", "3"}},
+                },
+                {
+                    "feature_col": ["num_citations"],
+                    "transform": {"name": "to_categorical", "separator": ","},
+                },
```

Comment:
We are adding these values to the config, but I don't see the logic of the tests changing. Shouldn't we be testing that the config is indeed parsed as expected?